accounts/models.py

```diff
@@ -47,6 +51,9 @@ def transfer(self, amount: decimal.Decimal, to_account: int) -> None:
         if type(amount) is not decimal.Decimal:
             amount: decimal.Decimal = decimal.Decimal(amount)
 
+        if amount < 0:
```

Comment:
Same suggestion as mentioned before about convert integer value into decimal value.

```suggestion
        if amount < decimal.Decimal(0.0):
```